src/main/java/com/hypherionmc/modpublisher/tasks/GithubUploadTask.java

```diff
@@ -109,15 +109,11 @@ public void upload() throws Exception {
                 return;
             }
 
-            if (!extension.getGithub().isCreateTag()) {
-                // FIXME It'd be nice to get tag by name, but GHRepository doesn't expose an API to do that
-                // For now, just iterate thorough all tags on the repo...
-                for (GHTag ghTag : ghRepository.listTags()) {
-                    if (tag.equals(ghTag.getName())) {
-                        project.getLogger().warn("Create tag for GitHub Release is disabled and tag {} already exists", tag);
-                        return;
-                    }
-                }
+            // FIXME this check doesn't make sense when `github.draft` is true,
+            //       because draft releases don't create a tag anyway...
```

Comment:
This API was broken and is marked "experimental". Should we take this opportunity to change behaviour for the _creating a draft_ scenario?

What do you think the best approach is?
- rename the option for clarity?
- just update the docs?
- add draft/public specific options?
- take a string/enum instead of a boolean?